src/psyclone/tests/nemo/nemo_psy_test.py

```diff
@@ -52,14 +52,6 @@
 BASE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),
```

Comment:
pylint complains that Fortran2003 is imported (line 47) but unused.

Reply:
deleted